egs/heroico/s5/local/heroico_answers_copy_wav_files.pl

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/perl -w
```

Comment:
is this file used? If not, can you please remove it (and the other files not used as well)?